docs/source/guides/adding-interactivity/state-as-a-snapshot/_examples/print_chat_message.py

```diff
@@ -16,24 +16,28 @@ async def handle_submit(event):
         print(f"Sent '{message}' to {recipient}")
 
     return html.form(
+        {"on_submit": handle_submit, "style": {"display": "inline-grid"}},
         html.label(
+            {},
```

Comment:
There's a handful of these empty dicts in this PR.

I recommend doing a search through this diff page for `{}`